README.md

```diff
@@ -0,0 +1,116 @@
+# Badgr
+
+![badgr](https://badgr.brigade2.io/v1/github/checks/brigadecore/badgr/badge.svg?appID=99005)
+[![codecov](https://codecov.io/gh/brigadecore/badgr/branch/main/graph/badge.svg?token=N1SQx2TZt0)](https://codecov.io/gh/brigadecore/badgr)
+[![Go Report Card](https://goreportcard.com/badge/github.com/brigadecore/badgr)](https://goreportcard.com/report/github.com/brigadecore/badgr)
+
+Badgr's creators love using [shields.io](https://shields.io/), for displaying
+various project statuses in our READMEs, but it doesn't support something we
+wanted badly-- the ability to generate a badge based on the results of a GitHub
+check suite associated with a specific GitHub App.
```

Comment:
Should we update the wording a bit to signify that associating to a specific GitHub App is optional?

Reply:
Additionally, when mentioning the use case of getting combined check suite statuses (appID not provided or '0'), should we mention (or link to) the method of determining status (e.g. deferring to highest severity)?

Reply:
> Should we update the wording a bit to signify that associating to a specific GitHub App is optional?

In this first paragraph, I really only wanted to address the impetus for creating our own utility instead of using shields.io directly.

I think the second paragraph covers the fact that app ID is optional, but please lmk if you think it's not adequate.

> should we mention (or link to) the method of determining status (e.g. deferring to highest severity)?

This seems worthwhile. Ok to do that particular bit in a follow-up?

Reply:
Ah, re-read the following paragraph; yup, looks good.  Follow-up fine as well 👍

Reply:
Created #2 to track.